app/views/layouts/application.html.haml

```diff
@@ -1,58 +1,70 @@
+= render 'application/head',  title: (content_for?(:title) ? "#{yield(:title)} | GOV.AU" : 'GOV.AU' )
+%body
+  %div.skip-to
+    %a{href: "#content"}
+      Skip to main content
+  %header{role: 'banner'}
+    = render 'application/signed_in_nav' if user_signed_in?
+    %section.prerelease-govau--header
+      - if content_for?(:header)
+        = yield(:header)
+      - else
+        .wrapper
+          %a.govau--logo{href: "/"}
+            = image_tag 'icons/logo-govau-full-2x.png', width: '270'
+            .brandbar.default
+              .b1
+              .b2
+          = #render partial: 'feedback_and_menu'
 
-          %nav.breadcrumbs--inverted{'aria-label' => 'breadcrumb'}
-            .wrapper
-              = breadcrumbs style: :ul, class: nil
+      %nav.breadcrumbs--inverted{'aria-label' => 'breadcrumb'}
+        .wrapper
+          = breadcrumbs style: :ul, class: nil
 
-    = yield(:after_header)
+= yield(:after_header)
 
-    %main
-      - if content_for?(:navbar)
-        = yield :navbar
-      = render 'application/alerts'
-      = yield
-    %footer{role: "contentinfo"}
-      .wrapper
-        %section
-          %figure
-            .footer-logo
-              = image_tag 'coat-of-arms.png', alt: 'Australian Government Coat of Arms'
-          .footer-links
-            %nav
-              %ul
-                %li
-                  %a{href: 'javascript:void(0)'} About
-                %li
-                  %a{href: 'javascript:void(0)'} Accessibility
-                %li
-                  %a{href: 'javascript:void(0)'} Copyright
-                %li
-                  %a{href: 'javascript:void(0)'} Help
-                %li
-                  %a{href: 'javascript:void(0)'} Site map
-                %li
-                  %a{href: 'javascript:void(0)'} Terms of use
-            %p
-              &copy; Commonwealth of Australia
-              %br
-              %a{href:'https://github.com/AusDTO/gov-au-ui-kit/blob/master/LICENSE.md'} Licensed under the MIT License.
+%main
+  - if content_for?(:navbar)
+    = yield :navbar
+  = render 'application/alerts'
+  = yield
+%footer{role: "contentinfo"}
+  .wrapper
+    %section
+      %figure
+        .footer-logo
+          = image_tag 'coat-of-arms.png', alt: 'Australian Government Coat of Arms'
+      .footer-links
+        %nav
+          %ul
+            %li
+              %a{href: 'javascript:void(0)'} About
+            %li
+              %a{href: 'javascript:void(0)'} Accessibility
+            %li
+              %a{href: 'javascript:void(0)'} Copyright
+            %li
+              %a{href: 'javascript:void(0)'} Help
+            %li
+              %a{href: 'javascript:void(0)'} Site map
+            %li
+              %a{href: 'javascript:void(0)'} Terms of use
+        %p
+          &copy; Commonwealth of Australia
+          %br
+          %a{href:'https://github.com/AusDTO/gov-au-ui-kit/blob/master/LICENSE.md'} Licensed under the MIT License.
```

Comment:
I know you didn't change this bit, but this is referencing the ui-kit licence file rather than ours. We might want to fix that.